### PR TITLE
remove duplicate includes

### PR DESCRIPTION
**Pull request by bvdberg**
Just remove the 2nd of duplicate includes.

### CODE REVIEW

File: arch/x86/include/ia32/kernel_arch_data.h [hunk 1]
```diff
@@ -52,8 +52,6 @@
 
 #ifndef _ASMLANGUAGE
 
-#include <sys/util.h>
```
Comment (galak) [inline arch/x86/include/ia32/kernel_arch_data.h]: Are we getting <sys/util.h> pulled in from another header?  don't see the duplicate here.

File: subsys/net/ip/net_shell.c [hunk 2]
```diff
@@ -59,7 +59,6 @@ LOG_MODULE_REGISTER(net_shell, LOG_LEVEL_DBG);
 #endif
 
 #if defined(CONFIG_NET_L2_PPP)
-#include <net/ppp.h>
```
Comment (galak) [inline subsys/net/ip/net_shell.c]: Hmm, wondering if we should remove the one above, and leave this one with ifdef protection.  @jukkar ?

**Review by galak — COMMENTED** [review]
LGTM, just a few minor comments/questions.

File: subsys/net/l2/canbus/6locan.c [hunk 3]
```diff
@@ -19,7 +19,6 @@ LOG_MODULE_REGISTER(net_l2_canbus, CONFIG_NET_L2_CANBUS_LOG_LEVEL);
 #include <sys/byteorder.h>
 #include <net/ethernet.h>
 #include <net/net_ip.h>
-#include <string.h>
```
Comment (alexanderwachter) [inline subsys/net/l2/canbus/6locan.c]: We are using memmove.

**Review by alexanderwachter — CHANGES_REQUESTED** [review]

File: subsys/net/l2/canbus/6locan.c (hunk 3 above)
Comment (galak) [reply subsys/net/l2/canbus/6locan.c]: I think the point is that <string.h> is included twice on lines 19 & 22.

Comment (alexanderwachter) [reply subsys/net/l2/canbus/6locan.c]: Oh, I didn't unfold.

**Review by alexanderwachter — APPROVED** [review]

**Comment by bvdberg** [comment]
I made changes in the commit message and signed it off as well. Do I need to close and reopen another pull request?

**Comment by stephanosio** [comment]
> I made changes in the commit message and signed it off as well. Do I need to close and reopen another pull request?

No, all the checks will be re-run once you force-push.

File: soc/arm/xilinx_zynqmp/soc.h [hunk 4]
```diff
@@ -13,7 +13,6 @@
 #ifndef _ASMLANGUAGE
 
 #include <device.h>
-#include <sys/util.h>
```
Comment (stephanosio) [inline soc/arm/xilinx_zynqmp/soc.h]: This change is okay.

In fact, it is getting removed in another PR anyway (#22740).

**Review by stephanosio — APPROVED** [review]

**Comment by stephanosio** [comment]
> 1: UC6 Body has no content, should at least have 1 line.

@bvdberg Your commit message should look like below:
```
generic: includes: remove duplicate includes

(Body with more detailed description about this commit)

Signed-off-by: Bas van den Berg <b.van.den.berg.nl@gmail.com>
```

File: subsys/net/ip/net_shell.c (hunk 2 above)
Comment (jukkar) [reply subsys/net/ip/net_shell.c]: Yes, better to remove the one above which is without ifdef protection.

**Comment by galak** [comment]
@bvdberg can you fix the gitlint issues and rebase?

**Review by andyross — APPROVED** [review]

**Review by joerchan — APPROVED** [review]

**Pull request closed without merging**